test/sandbox-test.js

```diff
@@ -72,65 +82,41 @@ describe("sinon.sandbox", function () {
         it("uses restorable clock", function () {
             this.sandbox.useFakeTimers();
 
-            assert.isFunction(this.sandbox.clock.restore);
+            assert.clock(this.sandbox.clock);
         });
 
-        it("passes arguments to sinon.useFakeTimers", function () {
-            var stub = sinon.stub(sinon, "useFakeTimers").returns({ restore: function () {} });
+        // Not sure how we can test this without monkey-patching or testing the implementation of lolex.
```

Comment:
When I stubbed `useFakeTimers` it cause a bunch of assertion failures in other modules, almost as if the clock stub was not being restored.  Let me try it again and see if I can get to the root of that issue.